coordinator/history/history_test.go

```diff
@@ -644,6 +644,61 @@ func TestHistory_WatchLatestTransitions(t *testing.T) {
 	}, 10*time.Millisecond, time.Millisecond)
 }
 
+func TestHistory_WalkTransitions(t *testing.T) {
+	h := &History{
```

Comment:
There shouldn't be shared state between test cases. Move into test case or make it a func.